Cabal/src/Distribution/Simple/Setup/Haddock.hs

```diff
@@ -457,7 +458,8 @@ data HaddockProjectFlags = HaddockProjectFlags
 defaultHaddockProjectFlags :: HaddockProjectFlags
```

Comment:
Why are `haddock-project` options defined in `Cabal` library at all.. I understand nothing to do with you but it's a `cabal-install` command.